Add gilt overlay capbility to ursula cli

lihkin213 opened the pull request.

lihkin213 committed:
Add gilt overlay capbility to ursula cli

Add argument to pass in the gilt config file location

epwn commented on ursula_cli/shell.py:
```diff
@@ -176,6 +176,21 @@ def _run_module(inventory, module, module_args, module_hosts='all',
+    if proc.returncode != 0:
+        print "WARNING: Overlay failed. Please check gilt configuration"
+    else:
+        print stdout
```
id recommend logging stderr here instead

epwn commented on ursula_cli/shell.py:
```diff
@@ -176,6 +176,21 @@ def _run_module(inventory, module, module_args, module_hosts='all',
+                            stderr=subprocess.PIPE)
+    stdout = proc.communicate()[0]
+    if proc.returncode != 0:
+        print "WARNING: Overlay failed. Please check gilt configuration"
```
id also recommend logging this in addition to printing it

epwn reviewed (approved):
Looks good to me. only thing i'd like to see but isnt a show stopper is to use the log in addition to print. (for the CI and such)

lihkin213 commented:
Updated to incorporate the comments around using LOG instead of print statements. Also added a new argument to allow passing in a gilt configuration file on the ursula command line.

epwn commented:
LGTM
